### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by ArtemKa2208**
work 5

### CODE REVIEW

File: inheritanceES6.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class User{
```
Comment (StanislawVictorovich) [inline inheritanceES6.js]: missed white space between class name and bracket

File: inheritanceES6.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      constructor(name,age){
```
Comment (StanislawVictorovich) [inline inheritanceES6.js:2]: the same here

File: inheritanceES6.js (hunk 1 above)
Comment (ArtemKa2208) [reply inheritanceES6.js]: done

File: inheritanceES6.js (hunk 2 above)
Comment (ArtemKa2208) [reply inheritanceES6.js:2]: done

**Review by StanislawVictorovich — APPROVED** [review]

**Pull request merged**